### PR TITLE
contrib/snibbetracker: new package (1.1.1)

### PR DESCRIPTION
**Pull request by z-erica**

### CODE REVIEW

**Comment by nekopsykose** [comment]
probably makes more sense to backport the whole patch? https://github.com/lundstroem/snibbetracker/commit/8483d24a61703bcbab114673fae11ca4c5f0ef28  

not sure about the other changes in it

File: contrib/snibbetracker/files/Makefile [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+INCLUDE_PATHS = -I../../snibbetracker/src/cJSON
+
+# Library paths
+# LIBRARY_PATHS = -LC:\Downloads\SDL2-devel-2.0.4-mingw\i686-w64-mingw32\lib
+
+# Compiler flags
+COMPILER_FLAGS = -Wall -std=gnu99 -lasound -Wno-unused-function $(CFLAGS)
+
+# Linker flags
+LINKER_FLAGS = -lSDL2 -lm $(LDFLAGS)
```
Comment (nekopsykose) [inline contrib/snibbetracker/files/Makefile:25]: and after you do that, put cflags in here too

File: contrib/snibbetracker/files/Makefile [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+%.o: %.c         # combined w/ next line will compile recently changed .c files
+	$(CC) -MD -MP $(COMPILER_FLAGS) $(INCLUDE_PATHS) -o $@ -c $<
+
+.PHONY : all     # .PHONY ignores files named all
+all: $(EXE)      # all is dependent on $(EXE) to be complete
+
+.PHONY : install
+install: all
+	install -D -m755 snibbetracker $(DESTDIR)$(PREFIX)/bin/snibbetracker
+	install -D ../demos/*.snibb -t $(DESTDIR)$(PREFIX)/share/lundstroem/snibbetracker/demos/
```
Comment (nekopsykose) [inline contrib/snibbetracker/files/Makefile:44]: you could also patch the -D here instead of manually installing

Comment (nekopsykose) [reply contrib/snibbetracker/files/Makefile:44]:
```
install -d $(DESTDIR)$(PREFIX)/bin/
install -m755 snibbetracker $(DESTDIR)$(PREFIX)/bin/
```

etc. can't do it in one

File: contrib/snibbetracker/template.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+pkgname = "snibbetracker"
+pkgver = "1.1.1"
+pkgrel = 0
+build_style = "makefile"
+make_cmd = "gmake"
+make_dir = "res/linux"
+makedepends = ["sdl-devel", "gmake"]
```
Comment (q66) [inline contrib/snibbetracker/template.py:7]: gmake->hostmakedepends

**Pull request merged**